modules/tidb/src/main/java/org/testcontainers/containers/TiDBContainer.java

```diff
@@ -0,0 +1,123 @@
+package org.testcontainers.containers;
+
+import org.jetbrains.annotations.NotNull;
+import org.testcontainers.containers.wait.strategy.HttpWaitStrategy;
+import org.testcontainers.utility.DockerImageName;
+
+import java.time.Duration;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * @author Icemap
+ */
+public class TiDBContainer extends JdbcDatabaseContainer<TiDBContainer> {
+
+    public static final String NAME = "tidb";
+
+    public static final String DOCKER_IMAGE_NAME = "pingcap/tidb";
```

Comment:
```suggestion
    private static final String NAME = "tidb";

    private static final String DOCKER_IMAGE_NAME = "pingcap/tidb";
```

Reply:
I will change the `NAME` and `TIDB_PORT` to private, but `DOCKER_IMAGE_NAME` is used in TiDBContainerProvider, so it can't be a private variable.